test/cli.js

```diff
@@ -6,7 +6,6 @@ var test = require('tap').test;
 global.Promise = require('bluebird');
 var getStream = require('get-stream');
 var figures = require('figures');
-var arrify = require('arrify');
```

Comment:
We don't need `arrify` because `Array.prototype.concat` already can concat single values
[Array.prototype.concat docs](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/concat)